build-support/bin/_generate_all_lockfiles_helper.py

```diff
@@ -128,6 +129,11 @@ def main() -> None:
             f"--coverage-py-extra-requirements={repr(CoverageSubsystem.default_extra_requirements)}",
             f"--coverage-py-interpreter-constraints={repr(CoverageSubsystem.default_interpreter_constraints)}",
             f"--coverage-py-lockfile={CoverageSubsystem.default_lockfile_path}",
+            # HCL2 for Terraform dependency inference
+            "--backend-packages=+['pants.backend.experimental.terraform']",
+            f"--terraform-hcl2-parser-version={TerraformHcl2Parser.default_version}",
+            f"--terraform-hcl2-parser-extra-requirements={repr(TerraformHcl2Parser.default_extra_requirements)}",
+            f"--terraform-hcl2-parser-lockfile={TerraformHcl2Parser.default_lockfile_path}",
```

Comment:
Needs to also set the --interpreter-constraints option. See black or isort in this file for an example.

Reply:
Done.